packages/flutter/lib/src/material/material.dart

```diff
@@ -617,27 +628,37 @@ abstract class InkFeature {
   /// Called when the ink feature is no longer visible on the material.
   final VoidCallback? onRemoved;
 
-  bool _debugDisposed = false;
+  bool _disposed = false;
```

Comment:
Why was the "debug" from this name removed?

Reply:
For some reason I needed to use `_disposed` to control a `markNeedsPaint` (as you have seen), but I just tested that it is not needed. I'll reverted it for now.